stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerDefaultCenterContent.kt

```diff
@@ -81,11 +81,9 @@ public class MessageComposerDefaultCenterContent : FrameLayout, MessageComposerC
      */
     private fun init() {
         binding = StreamUiMessageComposerDefaultCenterContentBinding.inflate(streamThemeInflater, this)
-        binding.messageEditText.addTextChangedListener(object : TextWatcherAdapter() {
-            override fun afterTextChanged(s: Editable) {
-                onTextChanged(s.toString())
-            }
-        })
+        binding.messageEditText.doAfterTextChanged { editable: Editable? ->
```

Comment:
TextWatcherAdapter is internal here and produces a warning

Reply:
Do we lose out on any functionality without it?

Reply:
`TextWatcherAdapter` is just a noop adapter:

```
@RestrictTo(LIBRARY_GROUP)
public class TextWatcherAdapter implements TextWatcher {

  @Override
  public void beforeTextChanged(@NonNull CharSequence s, int start, int count, int after) {}

  @Override
  public void onTextChanged(@NonNull CharSequence s, int start, int before, int count) {}

  @Override
  public void afterTextChanged(@NonNull Editable s) {}
}

```